polyschedule/image_tools/generate_algorythms.py

```diff
@@ -0,0 +1,191 @@
+import io
+from abc import ABC, abstractmethod
+
+from PIL import Image, ImageFont, ImageDraw
+
+import polyschedule.dates as dates
+
+
+class _FontStyle:
+    HEADING = ImageFont.truetype('./fonts/Comfortaa-Regular.ttf', 35)
+    LABEL = ImageFont.truetype('./fonts/Comfortaa-Light.ttf', 22)
+    NORMAL = ImageFont.truetype('./fonts/Comfortaa-Regular.ttf', 25)
```

Comment:
Probably it should be `DEFAULT` or `COMMON` instead of `NORMAL`

```suggestion
    DEFAULT = ImageFont.truetype('./fonts/Comfortaa-Regular.ttf', 25)
```